### ISSUE TITLE
Bento traceback error on Gatsby and other projects

### ISSUE THREAD

**Issue by pabloest** [returntocorp/bento#7]
Running Bento `0.1.0b2` on the [Gatsby.js project](https://github.com/gatsbyjs/gatsby/).

Bento gave me this error:

```
r2c.eslint: |██▋     | 00:12Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/bento/cli.py", line 110, in __tool_filter
    results = result.filter(tool.tool_id, __tool_findings(tool, config), baseline)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/bento/cli.py", line 94, in __tool_findings
    return tool.results(tool_config)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/bento/tool.py", line 149, in results
    violations = self.parser().parse(raw)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/bento/extra/eslint.py", line 63, in parse
    r["source"] = r["source"].split("\n")
KeyError: 'source'
  r2c.eslint: |        | 01:29
  r2c.flake8: |        | 01:29

✘ Error while running r2c.eslint:
'source'

✔ 0 findings in 89.01 s
```

**Comment by pabloest** [comment]
I got a very similar error when running Bento on Ansible.

```
 @pabloestrada  bento check
Running Bento checks...
  r2c.eslint: |██▋     | 00:01Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/bento/cli.py", line 110, in __tool_filter
    results = result.filter(tool.tool_id, __tool_findings(tool, config), baseline)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/bento/cli.py", line 94, in __tool_findings
    return tool.results(tool_config)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/bento/tool.py", line 148, in results
    raw = self.run(config, paths)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/bento/extra/eslint.py", line 178, in run
    result = self.exec(cmd, capture_output=True)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/bento/tool.py", line 126, in exec
    return subprocess.run(command, **new_args)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/subprocess.py", line 472, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/subprocess.py", line 775, in __init__
    restore_signals, start_new_session)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/subprocess.py", line 1522, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: './node_modules/eslint/bin/eslint.js': './node_modules/eslint/bin/eslint.js'
```

Bento kept running after that (`⚠ 4147 findings in 155.12 s`)

**Comment by pabloest** [comment]
A similar traceback error also showed on [Netlify CMS](https://github.com/netlify/netlify-cms).
```
 @pabloestrada  bento check
Running Bento checks...
  r2c.eslint: |        | 00:00Updating eslint...
npm WARN eslint-config-airbnb@18.0.1 requires a peer of eslint-plugin-import@^2.18.2 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-config-airbnb@18.0.1 requires a peer of eslint-plugin-jsx-a11y@^6.2.3 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-config-airbnb@18.0.1 requires a peer of eslint-plugin-react@^7.14.3 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-config-airbnb@18.0.1 requires a peer of eslint-plugin-react-hooks@^1.7.0 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-config-airbnb-base@14.0.0 requires a peer of eslint-plugin-import@^2.18.2 but none is installed. You must install peer dependencies yourself.

+ eslint@5.16.0
updated 1 package and audited 531 packages in 3.983s
found 0 vulnerabilities

  r2c.eslint: |██▋     | 00:05Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/bento/cli.py", line 110, in __tool_filter
    results = result.filter(tool.tool_id, __tool_findings(tool, config), baseline)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/bento/cli.py", line 94, in __tool_findings
    return tool.results(tool_config)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/bento/tool.py", line 149, in results
    violations = self.parser().parse(raw)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/bento/extra/eslint.py", line 63, in parse
    r["source"] = r["source"].split("\n")
KeyError: 'source'
  r2c.eslint: |        | 00:21
  r2c.flake8: |        | 00:21

✘ Error while running r2c.eslint:
'source'

✔ 0 findings in 21.89 s
```

**Comment by nbrahms** [comment]
Fixed in 0.1.0b3
